napari/benchmarks/benchmark_labels_layer.py

```diff
@@ -104,7 +109,16 @@ def time_get_value(self, n):
 
     def time_save_history(self, n):
         """Time to save history."""
-        self.layer._save_history()
+        current_slice = int(self.layer.coordinates[0])
```

Comment:
Can we avoid `self.layer.coordinates` here and just hard code this to something reasonable, we'll probably drop that attribute soon as we move the cursor information off the layer, so nice not to have here if not needed

Reply:
Definitely - realistically any valid slice will do, so I'll update that now.